calicoctl/calico_ctl/node.py

```diff
@@ -340,6 +342,9 @@ def _start_node_container_docker(ip, ip6, as_num, log_dir, node_image, detach, e
     calico_networking = os.getenv(CALICO_NETWORKING_ENV,
                                   CALICO_NETWORKING_DEFAULT)
 
+    calico_gobgp = os.getenv(CALICO_GOBGP_ENV,
+                             CALICO_GOBGP_DEFAULT)
+
```

Comment:
We should fail startup in calicoctl `if not calico_networking and calico_gobgp`.

However, this also has me wondering if we should refactor these options into a `--backend=<backend>` where `<backend>` can be `calico`, `none`, or `gobgp`. Would leave us more room to expand into future alternative backend sets that get added to calico-node, instead of constnalty adding environment variables that can't all be set at once. @tomdee does that sound reasonable?